Debugging.md

```diff
@@ -0,0 +1,83 @@
+# Debugging your Game Servers
+
+Many debugging tasks of PlayFab Multiplayer Servers can be done locally using [LocalMultiplayerAgent](https://github.com/PlayFab/LocalMultiplayerAgent). However, sometimes the local environment does not exactly match the actual service and you would like to connect directly to debug a running server because it is marked as unhealthy or not performing as expected.
```

Comment:
Instead of saying the local environment does not exactly match, I would recommend saying sometimes the behavior of the game server differs between the local environment and the actual service.